python/cudf/cudf/core/udf/utils.py

```diff
@@ -19,32 +21,52 @@
 
 import rmm
 
+from cudf._lib.strings_udf import (
+    column_from_udf_string_array,
+    column_to_string_view_array,
+)
 from cudf.core.column.column import as_column
+from cudf.core.dtypes import dtype
 from cudf.core.udf.masked_typing import MaskedType
+from cudf.core.udf.strings_typing import (
+    str_view_arg_handler,
+    string_view,
+    udf_string,
+)
 from cudf.utils import cudautils
 from cudf.utils.dtypes import (
     BOOL_TYPES,
     DATETIME_TYPES,
     NUMERIC_TYPES,
+    STRING_TYPES,
     TIMEDELTA_TYPES,
 )
 from cudf.utils.utils import _cudf_nvtx_annotate
 
+# Maximum size of a string column is 2 GiB
+_STRINGS_UDF_DEFAULT_HEAP_SIZE = os.environ.get(
+    "STRINGS_UDF_HEAP_SIZE", 2**31
+)
+heap_size = 0
+cudf_str_dtype = dtype(str)
```

Comment:
Should these be public? The second one looks like it should be a module constant.
```suggestion
_heap_size = 0
_CUDF_STR_DTYPE = dtype(str)
```